components/network/network.tf

```diff
@@ -1,5 +1,5 @@
 module "network" {
-  source = "git::https://github.com/hmcts/aks-module-network.git?ref=cft"
+  source = "git::https://github.com/hmcts/aks-module-network.git?ref=master"
```

Comment:
Can we confirm no differences between cft & master?

Reply:
I merged cft into my branch which was merged into master. cft is slightly behind master now https://github.com/hmcts/aks-module-network/compare/cft...master